### DISCUSSION TITLE
Is it possible to install `next-auth/react` only?

### DISCUSSION THREAD

**Discussion by p6l-richard** [nextauthjs/next-auth#5475] [Help]
I have a client-only app that depends on the authentication of a next app in my monorepo:
```
# monorepo
apps
 |- web
     |- A web app written in NextJS (Server & Client) that uses NextAuth
 |- extension
     |- A WebExtension written in Plasmo, React 18 (no server)
```

I'd like to use a custom version of `<SessionProvider />`, `signIn`, `signOut` in the `extension` app such that I can make it use the NextAuth configuration in `apps/web` under the hood. (with some additional behavior).

However, [the docs say](https://next-auth.js.org/configuration/options#options) that initialisation of options can only happen server-side?

I tried searching issues & discussion but was unable to find anything related, so I decided to open a new discussion. Let me know if that's not the right place for this. :) And awesome work on this, I appreciate it so much. 🙏

**Comment by TomFreudenberg** [comment]
Hi,

Could you please add some more details what you like to do and what should happen in the extension? I can't get what you want - Subject of title miss-leads to your question for me.

Have you seen the https://next-auth.js.org/getting-started/rest-api documentation? Maybe you could get something from that to access the authentication from within your extension?
